helfomat-infrastructure-batch/src/main/java/de/helfenkannjeder/helfomat/infrastructure/batch/ImportJobRunnerService.java

```diff
@@ -17,6 +18,7 @@
  * @author Valentin Zickner
  */
 @Service
+@PreAuthorize("hasRole('helfomat_admin')")
```

Comment:
I would suggest using @Secured("helfomat_admin") for simple role checks.
With a set of common roles, we can also think about convenient @interfaces like @isAdmin.

Reply:
fixed